ui/inspectors/calendar.reel/calendar-list.reel/calendar-list-day.reel/calendar-list-day.js

```diff
@@ -43,24 +43,40 @@ exports.CalendarListDay = Component.specialize(/** @lends CalendarListDay# */ {
         set: function(selectedDay) {
             if (this._selectedDay !== selectedDay) {
                 this._selectedDay = selectedDay;
-                if (selectedDay == this._day) {
+                if (selectedDay == this._object) {
                     this.element.scrollIntoView();
                 }
             }
         }
     },
 
+    hasEvents: {
+        value: null
+    },
+
+    _setHasEvents: {
+        value: function() {
+            if (this.displayedEvents.length) {
+                this.object.hasEvents = true;
+            } else {
+                this.object.hasEvents = false;
```

Comment:
You should rewrite it as `this.object._hasEvents = !!this.displayedEvents.length`.
(Please avoid creating new public properties on dataObjects)